### PR TITLE
cucumber-html-report

### PR DESCRIPTION
**Pull request by manuelq12**
Configuration of Cucumber-HTML-Report

image

### CODE REVIEW

File: report_generator.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+        metadata: {
+            "App Version":"0.0.1",
+            "Test Environment": "STAGING",
+            "Platform": "Windows 10",
+            "Executed": "Local"
+        }
+    };
+
+    reporter.generate(options);
+
```
Comment (aperdomob) [inline report_generator.js]: EOF

File: test/cucumber-test-json/cucumber_report.json [hunk 2]
```diff
@@ -0,0 +1,384 @@
+[
```
Comment (aperdomob) [inline test/cucumber-test-json/cucumber_report.json]: I think that this file should be not uploaded, should be generated into the ci process

**Review by aperdomob — CHANGES_REQUESTED** [review]

File: test/cucumber-test-json/cucumber_report.json (hunk 2 above)
Comment (manuelq12) [reply test/cucumber-test-json/cucumber_report.json]: Can you explain to me how to do this?
I placed the script for the json file to be created but it's still failing

File: report_generator.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+        reportSuiteAsScenarios: true,
+        scenarioTimestamp: true,
+        launchReport: true,
+        metadata: {
+            "App Version":"0.0.1",
+            "Test Environment": "STAGING",
+            "Platform": "Windows 10",
+            "Executed": "Local"
+        }
+    };
```
Comment (yesid-lopez) [inline report_generator.js]: indent

**Review by yesid-lopez — APPROVED** [review]

**Review by aperdomob — APPROVED** [review]

**Pull request merged**